### PR TITLE
Implemented pull to refresh feature

### LINKED ISSUE #41: Implement pull to refresh on all pages containing data load from server
Presently there is no page refreshing feature on the pages which display data from the server.
The pages are
- `EventAttendeesPage`
- `EventDashboardPage`
- `EventsPage`

`SwipeRefreshLayout` from android like feature is preferred.

### PR DESCRIPTION
**Pull request by mejariamol**
fixes #41

### CODE REVIEW

File: src/pages/event-attendees/event-attendees.ts [hunk 1]
```diff
@@ -56,6 +57,25 @@ export class EventAttendeesPage {
     });
   }
 
+  public doRefresh(refresher) {
+    this.attendeesService.loadAttendees(this.event.id).subscribe(
```
Comment (niranjan94) [inline src/pages/event-attendees/event-attendees.ts]: Let's not have duplication. Please do it in such that there is no duplication.

Comment (mejariamol) [inline src/pages/event-attendees/event-attendees.ts]: @niranjan94 `refresher.complete()` needs to be called after the async work. Thats why I couldn't make a common function. Please suggest how can be duplication avoided in this case?

Comment (niranjan94) [inline src/pages/event-attendees/event-attendees.ts]: You can pass a custom callback and execute it

Comment (mejariamol) [inline src/pages/event-attendees/event-attendees.ts]: @niranjan94 Avoided duplication. But not using callback. Please check if the approach is fine.

**Pull request merged**